src/Metadata/Property/Factory/ExtractorPropertyMetadataFactory.php

```diff
@@ -48,9 +48,15 @@ public function create(string $resourceClass, string $property, array $options =
             }
         }
 
+        try {
```

Comment:
Can't this entire block be replaced by a simple `interface_exists($resourceClass)` call?

Reply:
Fixed